### PR TITLE
Feature/initial code

### PR DESCRIPTION
**Pull request by andykitch**
Creates an initial next.js UI page with a map, created by geoJSON of country borders. The markers come from the nest.js graphQL backend, which is pointed at a postgres postGIS database.

### CODE REVIEW

File: API/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Base image
+FROM node:16
```
Comment (ray-millward-tessella) [inline API/Dockerfile:2]: We may want to consider a specific version of node so we don't get caught out by minor updates breaking things.

File: API/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+RUN npm install
+
+# Bundle app source
+COPY . .
+
+# Creates a "dist" folder with the production build
+RUN npm run build
+
+# Start the server using the production build
+CMD [ "node", "dist/main.js" ]
```
Comment (ray-millward-tessella) [inline API/Dockerfile:20]: When we've got testing and linting we might want to convert this to a multi-stage build but I don't know how that fits with github actions at this stage.

File: UI/api/api.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+
+const client = new ApolloClient({
+    uri: "/api/graphql",
+    cache: new InMemoryCache(),
```
Comment (ray-millward-tessella) [inline UI/api/api.tsx:5]: in memory caches can cause issues if you're trying to load updated data from the api on each request - we might need to consider turning this off later.

**Review by ray-millward-tessella — APPROVED** [review]
Looks good, we'll build on this over time.

File: API/Dockerfile (hunk 1 above)
Comment (andykitch) [reply API/Dockerfile:2]: Constraining it to 16.16.0

**Pull request merged**